### PR TITLE
[build] Migrate LLVM bots to Trusty

### LINKED ISSUE #2696: Use Travis Ubuntu Trusty 14.04 build env
If you just want to get started using our Trusty beta, then add the following to your `.travis.yml`.

```
sudo: required
dist: trusty
```

http://blog.travis-ci.com/2015-10-14-opening-up-ubuntu-trusty-beta/

### PR DESCRIPTION
**Pull request by tmpsantos**
LLVM is no longer hosting a debian repo and the build was broken. Trusty
has the LLVM version we need.

Fixes #2696

### CODE REVIEW

**Comment by tmpsantos** [comment]
:eyes: @brunoabinader @kkaefer

**Comment by brunoabinader** [comment]
Thank you @tmpsantos, this looks good! 👍

File: .travis.yml [hunk 1]
```diff
@@ -48,25 +52,23 @@ matrix:
...
       env: BUILDTYPE=Release _CXX=clang++-3.5 _CC=clang-3.5
       addons: *clang35
       script:
         - nvm install 4
         - nvm use 4
         - make node
         - make test-node
-      after_script:
-        - ccache --show-stats
-        - ./platform/node/scripts/after_script.sh ${TRAVIS_JOB_NUMBER} ${TRAVIS_TAG:-}
```
Comment (jfirebaugh) [inline .travis.yml]: Why did you remove the `after_script`? This is used to upload test-suite results to S3, so we can review them.

Comment (tmpsantos) [inline .travis.yml]: My bad, I thought it was the global default but I missed the `.../node/...` in the path. Gonna restore it.
